ocs_ci/utility/ibmcloud.py

```diff
@@ -413,16 +414,20 @@ def wait_for_volume_attach(self, volume):
                 instance, False otherwise
 
         """
-        cmd = f"ibmcloud is volume {volume} --output json"
-        out = run_ibmcloud_cmd(cmd)
-        out = json.loads(out)
 
-        if not out["volume_attachments"]:
-            logger.info("volume is not attached to node")
+        def is_volume_attached(volume):
```

Comment:
I prefer this nested function to be moved outside of this method this to be regular method as it can be used not just in this function but maybe others as well in future.

Also missing docstring and description of volume argument plus documentation of return value.

Reply:
Moved the method outside as regular method and added docstring